UnoCheck/CheckCommand.cs

```diff
@@ -315,6 +315,12 @@ public override async Task<int> ExecuteAsync(CommandContext context, CheckSettin
 							i--;
 					}
 				}
+				else if (!string.IsNullOrEmpty(diagnosis.Message))
+				{
+					// Display error/warning message when there's no suggestion
+					Console.WriteLine();
+					AnsiConsole.MarkupLine($"[bold {statusColor}]{statusEmoji} {diagnosis.Message}[/]");
+				}
```

Comment:
```suggestion
				else if (diagnosis.Status != Status.Ok && diagnosis.Message is { Length: > 0 } m)
				{
					// Display error/warning message when there's no suggestion
					Console.WriteLine();
					AnsiConsole.MarkupLine($"[bold {statusColor}]{statusEmoji} {m}[/]");
				}
```

Please also make sure to add tests for that.